python_modules/dagster/dagster/core/execution_plan/simple_engine.py

```diff
@@ -277,11 +277,19 @@ def _create_step_output_event(step_context, step_output_value, intermediates_man
             value=value,
         )
 
+        # This is a stopgap to facilitate writing tests against the value_repr exposed in the
```

Comment:
maybe make it its own function then a comment about what each case is doing exactly (and what error is being avoided) but this is an improvement